lighthouse-core/config/constants.js

```diff
@@ -54,6 +54,7 @@ const throttling = {
 const defaultSettings = {
   output: 'json',
   maxWaitForFcp: 30 * 1000,
+  maxWaitForLcp: 40 * 1000,
```

Comment:
This timing uses https://github.com/GoogleChrome/lighthouse/issues/11180#issuecomment-685931823 as a guideline but beyond that is pretty arbitrary. Is there a more definitive way to choose this timing?

Reply:
ah sorry @adamraine my comment was misleading. I don't think we actually need this to be a separate param compared to `maxWaitForLoad`, but I now that I'm looking at all the pieces, I kinda like the fine grained control and explicit it offers, so this WFM :)

Reply:
What do you think of the exact time value? 40s was basically a guess, would making the default value the same as `maxWaitForLoad` make more sense?